ui/app/templates/components/oidc/scope-form.hbs

```diff
@@ -4,9 +4,16 @@
       <ul>
         <li>
           <span class="sep">&#x0002f;</span>
-          <LinkTo @route={{concat "vault.cluster.access.oidc.scopes" (unless @model.isNew ".scope.details")}}>
```

Comment:
because of the timing of rending we had to break this out, otherwise it was causing transition failures.

Reply:
The new way is cleaner anyway, I like it 👍